pkg/dogstatsd/server.go

```diff
@@ -289,6 +298,42 @@ func NewServer(serverless bool) *Server {
 		}
 	}
 
+	// Creating a new logger
+	cfg := seelogCfg.NewSeelogConfig(
+		"dogstatsd",
+		"info",
+		"common",
+		"",
+		"%Date %Time | %Msg %n",
+		false)
+
+	// Configuring the log file path
+	logFile := config.Datadog.GetString("dogstatsd_log_file")
+	if logFile == "" {
+		logFile = constants.DefaultDogstatsDLogFile
+	}
+
+	// Configuring max roll for log file, if dogstatsd_log_file_max_rolls env var is not set (or set improperly ) within datadog.yaml then default value is 3
+	dogstatsd_log_file_max_rolls := config.Datadog.GetInt("dogstatsd_log_file_max_rolls")
+	if dogstatsd_log_file_max_rolls < 1 {
```

Comment:
I guess this goes down to whether we use our logic or seelog logic, only down side of using seelog logic i think is that if they change this specific logic in the future then our code might not work properly because of that edge case.

Reply:
How many files there will be on disk the agent runs with max_rolls=1?

Reply:
Changed logic to < 0